### DISCUSSION TITLE
Github改了名字，现在登录成了新用户了

### DISCUSSION THREAD

**Discussion by stevenli11** [BurnHair/docs#142] [技术支持]
我前几天Github改了名字，现在登录成了新用户，我的telegram还一直签到着，里面还有$39.38

**Comment by BennyThink** [comment]
两种办法：

1. 旧账号重置下密码（邮件沟通）
2. 旧账号余额转移到新账号（提供新旧账号的信息）

**Reply by stevenli11** [reply]
> 两种办法：
> 
> 1. 旧账号重置下密码（邮件沟通）
> 2. 旧账号余额转移到新账号（提供新旧账号的信息）


我一直用的github登录的，旧账号好像是leven1113来着，现在username改成了penglee13

**Reply by BennyThink** [reply]
好了
